src/clients/dotnet/docs.zig

```diff
@@ -0,0 +1,315 @@
+const builtin = @import("builtin");
+const std = @import("std");
+
+const Docs = @import("../docs_types.zig").Docs;
+const run = @import("../shutil.zig").run;
+const run_shell = @import("../shutil.zig").run_shell;
+const script_filename = @import("../shutil.zig").script_filename;
+const write_shell_newlines_into_single_line = @import("../shutil.zig").write_shell_newlines_into_single_line;
+
+pub const DotnetDocs = Docs{
+    .directory = "dotnet",
+
+    .markdown_name = "cs",
+    .extension = "cs",
+    .proper_name = ".NET",
+
+    .test_source_path = "",
+
+    .name = "tigerbeetle-dotnet",
+    .description = 
+    \\The TigerBeetle client for .NET.
+    \\
+    \\Note: The .NET client does not currently support M1/M2 Macs.
+    ,
+
+    .prerequisites = 
+    \\* .NET >= 6.0
```

Comment:
We support [.Net Standard 2.1](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1) as well, which includes .Net Core 3.1, Xamarin, Mono, and Unity.

Reply:
Ok, adjusted. Thank you!